NetTopologySuite.Samples.Console/Tests/Various/GraphBuilder.cs

```diff
@@ -1,137 +0,0 @@
-using System;

```

Comment:
Probably should have broken out "let's delete files that aren't actually in any of the projects" into their own step... oh well.